parcels/scripts/plottrajectoriesfile.py

```diff
@@ -71,7 +71,8 @@ def plotTrajectoriesFile(filename, mode='2d', tracerfile=None, tracerfield='P',
     if mode == '3d':
         from mpl_toolkits.mplot3d import Axes3D  # noqa
         plt.clf()  # clear the figure
-        ax = fig.gca(projection='3d')
+        # ax = fig.gca(projection='3d')
+        ax = fig.gca()
```

Comment:
I don't understand this change. I think this is what breaks the unit-tests. The `ax` is now not set as 3D anymore, so there is no `set_zlevel` method. Revert back to original?

Reply:
The change was necessary cause that line broke [bccb214](https://github.com/OceanParcels/parcels/pull/1247/commits/bccb2140274886f0ecbbd56dbfb019c036b2a3d1). In short: in Windows, the workflow obtains a newer version of matplotlib, and in matplotlib, the function `figure.gca()` has no `projection` parameters. Unexpected change, but therefore I needed to make this `projection` option part of the figure creation procedure. I guess help would be welcome from [bccb214](https://github.com/OceanParcels/parcels/pull/1247/commits/bccb2140274886f0ecbbd56dbfb019c036b2a3d1) onwards.